tools/privacyidea-export-linotp-counter.py

```diff
@@ -0,0 +1,107 @@
+                           Column('LinOtpTokenSerialnumber', Unicode(
+                               40), default=u'', unique=True, nullable=False,
+                                  index=True),
+                           Column(
+                               'LinOtpTokenType', Unicode(30), default=u'HMAC',
+                               index=True),
+                           Column(
+                               'LinOtpTokenInfo', Unicode(2000), default=u''),
+                           Column(
+                               'LinOtpTokenPinUser', Unicode(512), default=u''),
+                           Column(
+                               'LinOtpTokenPinUserIV', Unicode(32),
+                               default=u''),
+                           Column(
+                               'LinOtpTokenPinSO', Unicode(512), default=u''),
+                           Column(
+                               'LinOtpTokenPinSOIV', Unicode(32), default=u''),
+                           Column(
+                               'LinOtpIdResolver', Unicode(120), default=u'',
+                               index=True),
+                           Column(
+                               'LinOtpIdResClass', Unicode(120), default=u''),
+                           Column(
+                               'LinOtpUserid', Unicode(320), default=u'',
+                               index=True),
+                           Column(
+                               'LinOtpSeed', Unicode(32), default=u''),
+                           Column(
+                               'LinOtpOtpLen', Integer(), default=6),
+                           Column(
+                               'LinOtpPinHash', Unicode(512), default=u''),
+                           Column(
+                               'LinOtpKeyEnc', Unicode(1024), default=u''),
+                           Column(
+                               'LinOtpKeyIV', Unicode(32), default=u''),
+                           Column(
+                               'LinOtpMaxFail', Integer(), default=10),
+                           Column(
+                               'LinOtpIsactive', Boolean(), default=True),
+                           Column(
+                               'LinOtpFailCount', Integer(), default=0),
+                           Column('LinOtpCount', Integer(), default=0),
+                           Column(
+                               'LinOtpCountWindow', Integer(), default=10),
+                           Column(
+                               'LinOtpSyncWindow', Integer(), default=1000)
+                           )
+
+
+def get_linotp_uri(config_file):
+    with open(config_file) as f:
+        content = f.readlines()
+
+    lines = [l.strip() for l in content]
+    sql_uri = ""
+    for line in lines:
+        if line.startswith("SQLALCHEMY_DATABASE_URI"):
+            sql_uri = line.split("=")[1].strip().strip("'").strip('"')
```

Comment:
maybe we should apply the same fix here like below (`line.split("=", 1)...`) in order to be consistent.

Reply:
right. thx.